modules/rules_zig/0.5.0/presubmit.yml

```diff
@@ -0,0 +1,22 @@
+matrix:
+  platform: ["ubuntu2204", "macos"] # ["windows"]
+  bazel: [7.x, 8.x]
+tasks:
+  run_tests:
+    name: "Run tests"
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+      - "//..."
+bcr_test_module:
+  module_path: "e2e/workspace"
+  matrix:
+    platform: ["ubuntu2204", "macos"] # ["windows"]
+    bazel: [7.x, 8.x]
```

Comment:
https://buildkite.com/bazel/bcr-presubmit/builds/6540#01908e19-b063-40a9-b29a-3dc7f22ee1ab
```
2024/07/08 09:42:23 could not download Bazel: could not resolve the version '8.x' to an actual version number: unable to determine latest version: there are no releases available
```

Maybe you can use rolling if you want to test against the current HEAD.

Reply:
Sorry about that. I had added `8.x` here when I started to use rolling 8.x for rules_zig development and forgot about it for this PR. I've removed `8.x` here.

Reply:
Thanks!